docs/advanced-topics/advanced-losses-optimizers-and-poolers.md

```diff
@@ -0,0 +1,102 @@
+(advanced-losses-optimizers-poolers)=
+# {octicon}`mortar-board` Advanced Losses, Optimizers and Poolers
+Many of the models supported by Finetuner make use of similar methods throughout finetuning, such as the methods used for *calculating loss*, *sampling* and *pooling* . Finetuner offers alternatives to each of these methods, which can improve the performance of your finetuning run in some cases.
+
+## Loss Functions
+
+Loss functions are used to calculate the quality of embeddings while training; the higher the output of the loss function, the more the parameters of the model will be updated.
```

Comment:
```suggestion
Loss functions are used to calculate the quality of embeddings while training: The higher the output of the loss function, the larger the update to the parameters of the model.
```

I presume this is what you mean by "more updated": That the parameter change will be larger when the loss function output is larger.